docs/nifgen/attributes.rst

```diff
@@ -3677,6 +3938,187 @@ nifgen.Session properties
             - LabVIEW Property: **Triggers:Start:P2P Endpoint Fullness:Level**
             - C Attribute: **NIFGEN_ATTR_P2P_ENDPOINT_FULLNESS_START_TRIGGER_LEVEL**
 
+.. py:attribute:: p2p_endpoint_size
+
+    Returns the size, in samples per channel, of the peer-to-peer endpoint. This attribute is endpoint based.
+
+    The following table lists the characteristics of this property.
+
+    +----------------+-----------+
+    | Characteristic | Value     |
+    +================+===========+
+    | Datatype       | int       |
+    +----------------+-----------+
+    | Permissions    | read only |
+    +----------------+-----------+
+    | Channel Based  | False     |
+    +----------------+-----------+
+    | Resettable     | No        |
```

Comment:
Unrelated - I think we should remove this entry from the documentation table since it adds no value, just confusion, today.

Reply:
Removed